### ISSUE TITLE
bigflash and OTA with rom of 1MB

### ISSUE THREAD

**Issue by daffysub** [SmingHub/Sming#1118]
Hi,

I'm not sure to understand, because I didn't find detail about bigflash and support of OTA with 2 ROM of 1 MBytes.

You give an example with 1 ROM of 1 Mbytes and an other example with2 ROMs of 512kByte. But nothing for 2 ROM of 1 MBytes.

So, my questions are : 
- Is it possible to put 2 ROMs of 1 Mbytes (with a flash of 2 Mbytes minimum of course) and proceed to an OTA? 
- in makefile of rboot (Makefile-rboot.mk) we can see a reference to the rom1.ld file but were can we find it?

Thank you in adavance.

Best regard

**Comment by raburton** [comment]
Yes, you can do that. Two roms of 1mb is easy, you only need to link one rom file which can be flashed at any (or multiple) 1mb boundary/ies on the flash.  So you don't need rom1.ld. The Basic_rBoot sample does what you want and is already configured to run on 4mb flash (e.g. esp12).
